examples/rust_lang_tester/lang_tests/ignore.rs

```diff
@@ -1,4 +1,4 @@
-// ignore: this test is intentionally ignored
```

Comment:
So do we lose the ability to say why a test is ignored then?

I think forcing a reason was a good thing. I can barely remember why I did things last week, let alone why someone else did something 2 years ago :)

Reply:
See https://github.com/softdevteam/lang_tester/pull/120/commits/d09c0dc05a59dd7fb9cb58ab68fe8f2485ad5bd9 -- this PR makes things much more flexible!

Reply:
That's a human/per-project problem, not a lang_tester problem :)

Reply:
I'd argue that your change made it a human/per-project problem. Before you had to provide a reason. No ifs, not buts.

Perhaps if you use `ignore-if`, you have to also provide a `ignore-reason` line too?

Reply:
`ignore: <reason>` always had `<reason>` as optional, so this PR doesn't change the optionality. I don't want to mandate how people use this: indeed, the comment syntax is far more flexible than what we had before IMHO.

Reply:
I didn't realise that it was optional, so I'll back down.